hexa/plugins/connector_s3/tests/test_connector.py

```diff
@@ -0,0 +1,224 @@
+from django import test
+from django.urls import reverse
+from unittest import skip
+import json
+
+from hexa.catalog.models import CatalogIndex
+from hexa.plugins.connector_s3.models import (
+    Credentials,
+    Bucket,
+    BucketPermission,
+    Object,
+)
+from hexa.user_management.models import User, Team, Membership, Organization
+
+
+def query(client, payload):
+    return json.loads(
+        client.post(
+            "/graphql/",
+            json.dumps(
+                {
+                    "operationName": None,
+                    "variables": {},
+                    "query": payload,
+                }
+            ),
+            content_type="application/json",
+        ).content
+    )
+
+
+class ConnectorS3Test(test.TestCase):
```

Comment:
I would like to discuss this part with you too.

In a nutshell:

- I think we need to split the tests between something like `test_views`, `test_models` and `test_graph`
- The query method is going to be used a lot so we might as well have a `GraphQLTestCase` class with the `query` method
- It's not super important but I like using f-strings instead of interpolation